variables.tf

```diff
@@ -202,10 +202,11 @@ variable "configuration" {
     error_message = "Value for `configuration[\"archive_timeout\"]` must be 300 or more, if specified."
   }
 
-  validation {
-    condition     = var.configuration != null ? (var.configuration["wal_level"] != null ? contains(["replica", "logical"], var.configuration["wal_level"]) : true) : true
-    error_message = "Value for `configuration[\"wal_level\"]` must be either `replica` or `logical`, if specified."
-  }
+  # skip validation for issue #508 and #512
```

Comment:
suggest to link to https://github.com/terraform-ibm-modules/terraform-ibm-icd-postgresql/issues/512